NinjaResidence/DirectX.h

```diff
@@ -85,6 +85,12 @@ class DirectX {
 	* @brief �L�[�{�[�h���̓ǂݎ��
 	*/
 	void CheckKeyStatus();
+
+	/**
+	* @brief ���Âꂩ�̃L�[��������Ă��邩�̊m�F
+	* @return ����������Ă����true
+	*/
+	bool PushAnykey();
```

Comment:
これは無いわー。
けど「じゃあ何が良い？」と言われても困る系。
うーん、
```suggestion
	bool IsAnykeyPressed();
```
とかかなぁ。
https://qiita.com/ikngtty/items/3125c2210b8623a70311